jvm-packages/xgboost4j-spark/src/test/scala/ml/dmlc/xgboost4j/scala/spark/Utils.scala

```diff
@@ -16,15 +16,6 @@
 
 package ml.dmlc.xgboost4j.scala.spark
 
-import java.io.File
-
 trait Utils extends Serializable {
```

Comment:
It seems a bit ad-hock, since the number of CPUs is unrelated to sharing SparkContext. 

Actually, how about defaulting XGBoost to nWorkers equal to spark.task.cpus from SparkConfig?